test/rekt/features/knconf/data_plane.go

```diff
@@ -25,8 +25,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"knative.dev/eventing/test/rekt/resources/addressable"
 	"knative.dev/reconciler-test/pkg/eventshub"
+	. "knative.dev/reconciler-test/pkg/eventshub/assert"
 	"knative.dev/reconciler-test/pkg/feature"
-	eventshubmain "knative.dev/reconciler-test/pkg/test_images/eventshub"
 )
```

Comment:
what's the reason to replace the external (from `reconciler-test`) by adding more code here?

Reply:
the external is too hard to use.